.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "C_Cpp.errorSquiggles": "Disabled"
```

Comment:
Tego pliku nie powinno się wrzucać do repo. To ustawienia Visual Studio Code. Warto utworzyć sobie plik .gitignore i wpisać w nim
`.vscode`
aby ignorować wszystkie pliki z tego katalogu :)